unstructured_inference/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.7.19"  # pragma: no cover
+__version__ = "0.7.20-dev"  # pragma: no cover
```

Comment:
make at a real release and not a -dev one if planning to integrate with unstructured?